x/ethbridge/client/cli/tx.go

```diff
@@ -164,9 +179,9 @@ func GetCmdBurn(cdc *codec.Codec) *cobra.Command {
 func GetCmdLock(cdc *codec.Codec) *cobra.Command {
 	//nolint:lll
 	return &cobra.Command{
-		Use:   "lock [cosmos-sender-address] [ethereum-receiver-address] [amount] [symbol] --ethereum-chain-id [ethereum-chain-id]",
+		Use:   "lock [cosmos-sender-address] [ethereum-receiver-address] [amount] [symbol] [ceth-amount] [message-type] --ethereum-chain-id [ethereum-chain-id]",
```

Comment:
We need more help text around ceth-amount.  Is it the exact amount that's charged, or is it a maximum amount?

Reply:
it is maximum amount, I will update the usage.